src/ApplicationInsights/ApplicationInsights/ApplicationInsights/GetApplicationInsights.cs

```diff
@@ -15,10 +15,12 @@
 using Microsoft.Azure.Commands.ApplicationInsights.Models;
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 using System.Management.Automation;
 
 namespace Microsoft.Azure.Commands.ApplicationInsights
 {
+    [GenericBreakingChange("API version bump up to 2020-02, output type will be also updated to match the new API version", "2.0.0")]
```

Comment:
The API version contains date, does it? like 2020-02?

Reply:
fixed